source/adapters/native_cpu/enqueue.cpp

```diff
@@ -710,3 +710,11 @@ UR_APIEXPORT ur_result_t UR_APICALL urEnqueueWriteHostPipe(
 
   DIE_NO_IMPLEMENTATION;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urEnqueueNativeCommandExp(
+    ur_queue_handle_t, ur_exp_enqueue_native_command_function_t, void *,
+    uint32_t, const ur_mem_handle_t *,
+    const ur_exp_enqueue_native_command_properties_t *, uint32_t,
+    const ur_event_handle_t *, ur_event_handle_t *) {
+  DIE_NO_IMPLEMENTATION;
```

Comment:
We are trying to get rid of `DIE_NO_IMPLEMENTATION`, could you use `UR_RESULT_ERROR_UNSUPPORTED_FEATURE` here?

Reply:
Change made, thanks.